native/src/main/java/io/ballerina/persist/cmd/Generate.java

```diff
@@ -100,24 +106,29 @@ public void execute() {
             return;
         }
         try {
-            ArrayList<Entity> entityArray = readBalFiles();
+            Map<String, ArrayList> retMap = readBalFiles();
+            ArrayList<Entity> entityArray = retMap.get("metaDataArray");
+            ArrayList<ModuleMemberDeclarationNode> returnModuleMembers = retMap.get("entityMembers");
```

Comment:
Make map keys as constants

Reply:
Updated in [a7041fd](https://github.com/ballerina-platform/persist-tools/pull/31/commits/a7041fdbccf9572781cd185694e8cb15352d4e0b) as a new object to store the values